validator_client/src/block_service.rs

```diff
@@ -346,6 +351,11 @@ impl<T: SlotClock + 'static, E: EthSpec> BlockService<T, E> {
                                 &metrics::BLOCK_SERVICE_TIMES,
                                 &[metrics::BEACON_BLOCK_HTTP_GET],
                             );
+                            info!(
```

Comment:
I think we need to put this log under the `get_validator_blocks` (two lines below), since we haven't received the block until that call returns `Ok`.

Reply:
I just moved it under the match statement entirely. Hopefully that's fine with you.

Reply:
Oh yeah, good idea!